src/components/ListItemLarge.tsx

```diff
@@ -24,7 +24,11 @@ export const ListItemLargeStyles = {
 
 const styles = StyleSheet.create(ListItemLargeStyles);
 
-const withStyle = (props: ListItemProps) => {
+interface ListItemLargeProps extends ListItemProps {
+  height?: number;
```

Comment:
this component should have been left untouched by this change. There is a new component on our design system called `ListItemComparator`, which should be implemented as a component in react. The difference between `ListItem` and `ListItemLarge` is its `height`, so moving that to a prop makes little sense.

The idea of the design system is to hide these choices from the user of the DS itself, so that you will never have `height={96}` on the lib user side, you will simply have to pick one of the available components (`ListItem` vs `ListItemLarge` vs `ListItemComparator`). At the same time, we want to prevent the lib user from shooting themselves in the foot. What happens if they set, e.g., `height={500}`?